devtools-ci-index.yaml

```diff
@@ -2118,7 +2118,7 @@
           secrets:
             - *quay-eclipse-che-credentials
     triggers:
-        - timed: '*/30 * * * *'
+        - timed: '59 23 * * *'
```

Comment:
Just wondering why you opt for "At 23:59" - https://crontab.guru/#59_23_*_*_*  ?
Anyway, I believe this change is not needed since https://github.com/openshiftio/openshiftio-cico-jobs/pull/1039 was merged today